app/build.gradle

```diff
@@ -34,7 +34,7 @@ android {
 }
 
 dependencies {
-    implementation fileTree(dir: 'libs', include: ['*.jar'])
+    implementation fileTree(include: ['*.jar'], dir: 'libs')
```

Comment:
What's the difference this swapping makes?

Reply:
As you know, I had troubles when integrating the SDK into the app :(. After adding the jars as a library through the IDE this dependency declaration has changed.